cmd/_integration-tests/transport/http_test.go

```diff
@@ -606,7 +607,21 @@ func testHTTP(
 		t.Fatal(errors.Wrap(err, "cannot make http request"))
 	}
 
-	err = json.Unmarshal(respBytes, &resp)
+	switch v := resp.(type) {
+	case *pb.GetWithQueryResponse:
```

Comment:
You cannot:
```
./http_test.go:612: cannot use v (type interface {}) as type proto.Message in argument to jsonpb.UnmarshalString:
        interface {} does not implement proto.Message (missing ProtoMessage method)
```

By being only one type per case then the type of `v` is known. If you combine them into one then `v` is of type `interface{}`.